pyls/plugins/definition.py

```diff
@@ -6,8 +6,10 @@
 
 
 @hookimpl
-def pyls_definitions(document, position):
-    definitions = document.jedi_script(position).goto_assignments()
+def pyls_definitions(config, document, position):
+    expected_settings = {'follow_imports', 'follow_builtin_imports'}
+    params = {k: v for k, v in config.settings().items() if k in expected_settings}
```

Comment:
Config should be namespaced by the plugin, e.g. `pyls.plugins.jedi_definition` https://github.com/palantir/python-language-server/blob/develop/vscode-client/package.json#L38